layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -559,6 +559,17 @@ in a split window to the right."
   (split-window-horizontally)
   (other-window 1))
 
+(defun spacemacs/layout-grid ()
```

Comment:
I'm not sure if the name of the function should include a `2x2` or this is ok.

Reply:
It's OK as is, I don't realy think other grid layouts might be practical due to the small size of windows.

Sometimes I code with a 2x2 layout with an extra neotree open on the side and it becomes hard to read if I have to do many changes in just one file. But it's great for small one/two liners across multiple files.

Reply:
Yup ok, that's my way of thinking too :)